app/src/components/alarm-clock/alarm-clock.js

```diff
@@ -1,75 +1,75 @@
-import { Component } from "react";
+import { Component } from 'react'
 
-import StorageService from "../../service/storageService";
+import StorageService from '../../service/storageService'
 
 class AlarmClock extends Component {
-  constructor() {
-    super();
-    //window.Neutralino.computer.getMemoryInfo();
+  constructor () {
+    super()
+    // window.Neutralino.computer.getMemoryInfo();
 
     this.state = {
-      currentTime: "",
-      alarms: [],
-    };
+      currentTime: '',
+      alarms: []
+    }
     this.loadAlarmsFromDb()
-    this.setAlarmTime = this.setAlarmTime.bind(this);
+    this.setAlarmTime = this.setAlarmTime.bind(this)
   }
 
-  async loadAlarmsFromDb() {
-    const config = await StorageService.loadConfig();
+  async loadAlarmsFromDb () {
+    const config = await StorageService.loadConfig()
     this.setState({
-        alarms: config.alarms,
-      });
+      alarms: config.alarms
+    })
   }
 
-  setAlarmTime(event) {
-    event.preventDefault();
-    const inputAlarmTimeModified = event.target.value + ":00";
+  setAlarmTime (event) {
+    event.preventDefault()
+    const inputAlarmTimeModified = event.target.value + ':00'
     this.setState({
-      alarms: [...this.state.alarms, inputAlarmTimeModified],
-    });
+      alarms: [...this.state.alarms, inputAlarmTimeModified]
+    })
   }
 
-  checkAlarmClock() {
+  checkAlarmClock () {
     this.loadAlarmsFromDb()
     this.state.alarms.forEach((alarm) => {
-      if (this.state.currentTime === alarm + ":00") {
-        console.log("alarme aki!");
+      if (this.state.currentTime === alarm + ':00') {
+        console.log('alarme aki!')
         window.Neutralino.os.showNotification(
-          "Não esqueça de bater o seu ponto",
-          "Descanço merecido, não esquece de bater o ponto",
-          "INFO"
-        );
+          'Não esqueça de bater o seu ponto',
+          'Descanço merecido, não esquece de bater o ponto',
+          'INFO'
+        )
       }
-    });
+    })
   }
 
-  componentDidMount() {
-    this.clock = setInterval(() => this.setCurrentTime(), 1000);
-    this.interval = setInterval(() => this.checkAlarmClock(), 1000);
+  componentDidMount () {
+    this.clock = setInterval(() => this.setCurrentTime(), 1000)
+    this.interval = setInterval(() => this.checkAlarmClock(), 1000)
   }
 
-  componentWillUnmount() {
-    clearInterval(this.clock);
-    clearInterval(this.interval);
+  componentWillUnmount () {
```

Comment:
Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/ruizivo/ahgora-login/pullRequest?prid=9873926)
```suggestion
  componentWillUnmount() {
```